lib/embulk/output/google_spreadsheets.rb

```diff
@@ -50,9 +53,11 @@ def init
         @mode        = task["mode"].to_sym
         @row         = task["row_index"]
         @col         = task["col_index"]
-        @null_representation = task["null_representation"]
+        @null_string = task["null_string"]
 
         @worksheet = self.class.build_worksheet_client(task)
+
+        Time.zone = task["default_timezone"]
```

Comment:
This is very very bad idea. `Time` is shared among all plugins and embulk itself.